README.md

```diff
@@ -26,7 +26,7 @@ cd vibeline
 2. Create and activate a virtual environment:
 ```bash
 python -m venv vibenv
-source vibenv/bin/activate  # On Windows: vibenv\Scripts\activate
+[ -d "vibenv" ] && source vibenv/bin/activate  # On Windows: vibenv\Scripts\activate
```

Comment:
Updated this to only use the python venv if its defined. This is necessary for the docker image since it does not use a virtual environment